__tests__/app.test.js

```diff
@@ -175,21 +175,67 @@ describe('6. GET /api/reviews/:review_id/comments', () => {
 
 describe('7. POST /api/reviews/:review_id/comments', () => {
   test('status:201, responds with comment newly added to the database', () => {
+    const REVIEW_ID = 2;
     const newComment = {
       author: 'mallionaire',
       body: 'rubbish mate, grumble!',
+      review_id: REVIEW_ID
```

Comment:
The ticket states the only things on the request body are the author and body properties. So we don't/shouldn't need to include the review ID here - it is in the URL as a parameter (and can be accessed by your controller on req.params).